modules/user/src/main/Cached.scala

```diff
@@ -91,6 +91,12 @@ final class Cached(
     expireAfter = _.ExpireAfterWrite(10 seconds)
   )
 
+  val top50OnlineIds = asyncCache.single[List[User.ID]](
+    name = "user.top50onlineid",
+    f = UserRepo.idsByIdsSortRating(onlineUserIdMemo.keys, 50),
+    expireAfter = _.ExpireAfterWrite(10 seconds)
+  )
+
```

Comment:
Oops, earlier I was tripped up by `flatMap` syntax & strange compiler errors which have since subsided.  I've fixed this in the next commit (now pushed).